app/controllers/ApiController.scala

```diff
@@ -1,28 +1,27 @@
 package controllers
 
-import java.io.{OutputStream, PipedInputStream, PipedOutputStream}
-
 import akka.stream.scaladsl.{Source, StreamConverters}
 import akka.util.ByteString
-import javax.inject.Inject
-import play.api.Logging
-import play.api.mvc._
-import play.api.libs.json._
-import play.api.libs.json.Reads._
-import java.nio.file.Paths
-import java.time.format.DateTimeFormatter
-import java.time.LocalDateTime
-
-import scala.concurrent.{ExecutionContext, Future}
 import controllers.auth.AuthActionFactory
 import models.FeatureToggleModel.FeatureToggleService
 import models._
 import models.config.SmuiVersion
-import models.input.{InputTagId, InputValidator, ListItem, SearchInputId, SearchInputWithRules}
+import models.input._
```

Comment:
Can you check your IDE?   We typically have been putting ALL the imports in, isntead of using the short cut...   This make a lot of changes on this, which makes it harder to review.

Reply:
Reverted this import changes